assets/sass/components/surveys/_googlesitekit-surveys.scss

```
@@ -69,6 +69,16 @@
 			padding-top: 12px;
 			text-align: left;
 		}
+
+		.googlesitekit-survey__terms {
+			margin: 0 0 -12px;
+			padding-top: 12px;
+
+			a[target="_blank"] {
```

We do have a `.googlesitekit-cta-link--external` class we can target here instead of this selector, which I _think_ is a bit more efficient.
```suggestion
			.googlesitekit-cta-link--external {
```

---

Updated.